src/pages/jisuOh/Main/Comment/Comment.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import './Comment.scss';
+
+class Comment extends React.Component {
+  state = {
+    isClicked: false,
+    imagesPath: {
+      clickHeart: 'images/jisuOh/heart-red.png',
+      clickRed: 'images/jisuOh/heart.png',
+    },
+  };
+
+  toggleImage = e => {
+    const { isClicked } = this.state;
+    this.setState({ isClicked: !isClicked });
```

Comment:
setState할 때 기존의 state값을 기반으로 새 state값을 만들어야 하는 경우에는 이렇게 this.state를 통해서 가져와도 되지만, 
setState의 인자로 객체가 아니라 함수를 넣으면 인자로 prevState를 가져옵니다. 기존 state값 기반으로 변화시킬때는 이렇게 인자로 받아서 사용하는게 외부 요인에 영향을 받지 않을 수 있어서 좀 더 권장되는 방법입니다!
```suggestion
    this.setState((prevState) => {return{isClicked:!prevState.isClicked}});
```
좀 더 간편하게 표현하면
```suggestion
    this.setState((prevState) => ({isClicked:!prevState.isClicked}));
```

Reply:
와 저렇게도 표현할 수 있군요! 새로운 방법을 알게 됐네요~ 감사합니다!